### DISCUSSION TITLE
JustJ @ maven central?

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-justj/justj#8] [Ideas]
P2 site is great, but only consumable by P2, but most tools support maven repositories, is there any plan to distribute JustJ as a maven artifact?

**Comment by merks** [comment]
There are no plans to publish Maven artifacts.   It would be easy (almost no effort) to create a Maven compatible repository using the CBI Aggregator as I do for EMF here:

https://ci.eclipse.org/emf/job/maven-publish/lastSuccessfulBuild/artifact/

But probably you mean publish to maven central...

**Reply by laeubi** [reply]
Yes upload to maven central would be quite good, but maybe this is more something for the Adoptium project if that is the source?

I always though it would be great to reference a JVM as part of a maven coordinate instead of using different (specific) APIs just to download the binary file...

**Comment by merks** [comment]
Providing a full JDK is might also be useful too.

**Reply by laeubi** [reply]
Maven repositor has the notation of "classifiers", so for example we have `org.eclipse.justj.openjdk.hotspot` as group id, and `jre.minimal.linux.x86_64` as artifactid one can then deploy for version `11.0.18.v20230129-1042`
classifier = jdk
classfier = jre

... just in case that what you mean.

**Comment by laeubi** [comment]
It seems there are some demands already, but it was not yet considered, really sad:
- https://github.com/AdoptOpenJDK/TSC/issues/82
